docs/data/joy/components/linear-progress/linear-progress.md

```diff
@@ -0,0 +1,83 @@
+---
+product: joy-ui
+title: React Linear Progress component
+githubLabel: 'component: LinearProgress'
+---
+
+# Linear Progress
+
+<p class="description">Linear Progress indicators, commonly known as loaders, express an unspecified wait time or display the length of a process.</p>
+
+## Introduction
+
+Progress indicators inform users about the status of ongoing processes, such as loading an app, submitting a form, or saving updates.
+
+The `LinearProgress` is indeterminate by default, indicating an unspecified wait time.
+To actually have it represent how long an operation will take, use the [determinate](#determinate) mode.
+
+The animations of the components rely on CSS as much as possible to work even before the JavaScript is loaded.
+
+{{"demo": "LinearProgressUsage.js", "hideToolbar": true}}
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Component
+
+After [installation](/joy-ui/getting-started/installation/), you can start building with this component using the following basic elements:
+
+```jsx
+import LinearProgress from '@mui/joy/LinearProgress';
+
+export default function MyApp() {
+  return <LinearProgress />;
+}
+```
+
+### Variants
+
+The linear progress component supports the four global variants: `solid`, `soft` (default), `outlined`, and `plain`.
+
+{{"demo": "LinearProgressVariants.js"}}
+
+### Colors
+
+Every palette included in the theme is available via the `color` prop.
+Play around combining different colors.
+
+{{"demo": "LinearProgressColors.js"}}
+
+### Sizes
+
+The linear progress component comes with three sizes out of the box: `sm`, `md` (the default), and `lg`.
+
+{{"demo": "LinearProgressSizes.js"}}
+
+:::success
+To learn how to add more sizes to the component, check out [Themed components—Extend sizes](/joy-ui/customization/themed-components/#extend-sizes).
+:::
+
+### Determinate
+
+You can use the `determinate` prop if you want to indicate a specified wait time.
+
+{{"demo": "LinearProgressDeterminate.js"}}
+
+### Thickness
+
+Provides a number to `thickness` prop to control the bar's stroke width.
+
+{{"demo": "LinearProgressThickness.js"}}
+
+## CSS variables
+
+Play around with all the CSS variables available on the component to see how the design changes.
+
+You can use those to customize the component on both the `sx` prop and the theme.
+
+{{"demo": "LinearProgressVariables.js", "hideToolbar": true}}
+
+## Common examples
+
+### With label
+
+{{"demo": "LinearProgressWithLabel.js", "bg": true}}
```

Comment:
```suggestion
{{"demo": "LinearProgressWithLabel.js"}
```

This is all looking great, Benny! Just a quick suggestion to maybe have this demo bg consistent with all the others? Would it have a specific reason why this one has it set to true?

Reply:
I edited that one. I thought that it could show a different way of customization because `With label` is under the common example:

<img width="788" alt="image" src="https://user-images.githubusercontent.com/18292247/197112879-c792ffb8-82dc-4e11-8aad-6058ff889d7f.png">

Anyway, not a hard preference for me. I also looks great without the "bg".

Reply:
Yeah, I think I generally prefer with no background. In fact, I have a PR tackling these backgrounds a bit, so they'll hopefully look a bit better 😬

Reply:
I think we can remove the background. On my way!